spread.yaml

```diff
@@ -32,19 +42,16 @@ suites:
         summary: Integration tests
 
 prepare: |
-    DEBS="git lxd kvm"
-    if ! dpkg -l $DEBS; then
-        apt-get update
-        apt-get install -y $DEBS
-    fi
+    apt update
+    apt install -y git lxd qemu-kvm tree
 
     # Cache is only available if SEND_CACHE=1 is set.
-
-    # Cache image with: mkdir tests/cache/lxd-ubuntu-16.04 && lxc export ubuntu:16.04 tests/cache/lxd-ubuntu-16.04
-    if ! lxc image list | grep '16\.04'; then
-        lxc image import tests/cache/lxd-ubuntu-16.04/meta-* tests/cache/lxd-ubuntu-16.04/[a-f0-9]* os=ubuntu version=16.04 architecture=amd64 remote=ubuntu
-    fi
-
+    mkdir -p tests/cache
+    
+    # Start lxd instance
+    lxd init --auto
+    lxc launch ubuntu:16.04
```

Comment:
The original logic was building a cache and allowing one to use an existing cache. The new one has blown that away. It would be good to preserve the ability to use a cached image, and test it to make sure it still works.  Otherwise every time we run tests with qemu, we'll be paying the price of redownloading an image.

Reply:
Sure, I'll restore the cache.
Thanks

Reply:
Does it still work?

Reply:
@niemeyer  Yes, it is working